packages/dd-trace/src/opentracing/span.js

```diff
@@ -145,6 +145,18 @@ class DatadogSpan {
     return this._spanContext._baggageItems[key]
   }
 
+  getAllBaggageItems () {
+    return this._spanContext._baggageItems
```

Comment:
These 3 functions LGTM. My only question is, should we return `_baggageItems` directly here, or instead return an immutable copy, or something like an iterator? The OpenTelemetry spec says we should, but I don't think we necessarily need to follow that in the this API. Are there any concerns with exposing the internal `_baggageItems` to users? This may vary between languages.

Reply:
good idea, I'll return an immutable copy to be safe